### PR TITLE
Angular changelog updates for alpha+2

### PR DESCRIPTION
**Pull request by chalin**

### CODE REVIEW

File: src/angular/changelog.md [hunk 1]
```diff
@@ -52,21 +52,30 @@ but you can see a preview at
...
-because the router and forms implementations are moving into new packages
-(`angular_router` and `angular_forms`).
-We also expect to remove references to `resolved_identifiers`.
+because the forms implementation is moving into a new package (`angular_forms`).
 
 * Updated references to packages and transformers in `pubspec.yaml`:
   * `angular2` &rarr; `angular`
-* Updated `angular` <!-- & angular_components --> version:
-  * `^3.1.0` &rarr; `^4.0.0-alpha`
+  * Removed `reflection_remover` transformer entry
```
Comment (kwalrath) [inline src/angular/changelog.md]: Shouldn't we add an item about `resolved_identifiers`? (e.g. https://github.com/dart-lang/site-webdev/pull/924/files#diff-bf0dd29f2947b74d8be0ab4589d9c633)

File: src/angular/changelog.md [hunk 2]
```diff
@@ -179,3 +192,5 @@ More information:
 * [PR #478](https://github.com/dart-lang/site-webdev/pull/478/files) (initial text and toh-0 tests)
 * [PR #567](https://github.com/dart-lang/site-webdev/pull/567/files?w=1) (toh-6 tests)
 
+[`angular` changelog]: https://pub.dartlang.org/packages/angular/versions/4.0.0-alpha%2B2#pub-pkg-tab-changelog
```
Comment (kwalrath) [inline src/angular/changelog.md]: We should use these for the links at the top of this page, too.

File: src/angular/changelog.md [hunk 3]
```diff
@@ -52,21 +52,30 @@ but you can see a preview at
 All pubspecs and imports changed, as well as API doc URLs,
 due to the `angular2` package changing its name to `angular`.
 We expect imports (and API doc URLs) to change again before 4.0 is stable,
-because the router and forms implementations are moving into new packages
-(`angular_router` and `angular_forms`).
-We also expect to remove references to `resolved_identifiers`.
+because the forms implementation is moving into a new package (`angular_forms`).
```
Comment (kwalrath) [inline src/angular/changelog.md]: expect imports -> expect some imports

Comment (chalin) [inline src/angular/changelog.md]: done

File: src/angular/changelog.md (hunk 1 above)
Comment (chalin) [inline src/angular/changelog.md]: Ah yes, I added that to the gulp task but forgot to add it here. Good catch! Done.

File: src/angular/changelog.md (hunk 2 above)
Comment (chalin) [inline src/angular/changelog.md]: done

**Comment by chalin** [comment]
@kwalrath - in the last commit I also adjusted the 4.x change list ordering (putting package version changes first) and tweaked some of the text. PTAL. I'll merge now, but we can address any further tweaks later.

**Comment by kwalrath** [comment]
Your changes look great.

**Pull request merged**